src/main/java/com/jagrosh/jmusicbot/commands/MusicCommand.java

```diff
@@ -73,6 +73,13 @@ protected void execute(CommandEvent event)
                 event.replyError("You must be listening in "+(current==null ? "a voice channel" : "**"+current.getName()+"**")+" to use that!");
                 return;
             }
+
+            if(userState.getGuild().getAfkChannel().equals(userState.getChannel()))
```

Comment:
I think this will NPE if no afk channel is set

Reply:
```suggestion
VoiceChannel afkChannel = userState.getGuild().getAfkChannel();
if(afkChannel != null && afkChannel.equals(userState.getChannel()))
```

Reply:
Oh, right! Also, thank you @Sanduhr32, I've applied your suggestion.

Reply:
The issue is just it didn't let me do indention. Would you please do that?

@jagrosh I suggest squash merging this pr.